assets/lang/en_US.ini

```diff
@@ -539,13 +521,11 @@ FullScreen = Fullscreen
 Hack Settings = Hack settings (may cause glitches)
 Hardware Tessellation = Hardware tessellation
 Hardware Transform = Hardware transform
-hardware transform error - falling back to software = Hardware transform error, falling back to software
```

Comment:
Still used in GLES and Direct3D 9:
https://github.com/hrydgard/ppsspp/blob/287dd66f20f2a03acd316a6f5d5b98ff2eb6ad26/GPU/Directx9/ShaderManagerDX9.cpp#L594

-[Unknown]